--- Makefile	2013-09-18 14:54:17.331006173 -0700
+++ Makefile.patched	2013-09-18 14:54:24.907005876 -0700
@@ -6,10 +6,10 @@
 	./check-local $(IT)
 
 runit: load runit.o unix.a byte.a
-	./load runit unix.a byte.a -static
+	./load runit unix.a byte.a
 
 runit-init: load runit-init.o unix.a byte.a
-	./load runit-init unix.a byte.a -static
+	./load runit-init unix.a byte.a
 
 runsv: load runsv.o unix.a byte.a time.a
 	./load runsv unix.a byte.a time.a
